solr/core/src/java/org/apache/solr/handler/admin/CoreAdminHandler.java

```diff
@@ -546,7 +553,6 @@ private void addTask(TaskObject taskObject) {
       }
 
       taskObject.status = RUNNING;
```

Comment:
Isn't it no longer necessary to mark as RUNNING here since it's already done so in the lambda?

Reply:
Thanks. Forgot once again to remove old line :facepalm: